server/coordinator/node_picker.go

```diff
@@ -4,53 +4,91 @@ package coordinator
 
 import (
 	"context"
-	"strconv"
 	"time"
 
 	"github.com/CeresDB/ceresmeta/server/cluster/metadata"
 	"github.com/CeresDB/ceresmeta/server/hash"
 	"github.com/CeresDB/ceresmeta/server/storage"
 	"github.com/pkg/errors"
+	"github.com/spaolacci/murmur3"
 	"go.uber.org/zap"
 )
 
 type NodePicker interface {
-	PickNode(ctx context.Context, shardID storage.ShardID, registerNodes []metadata.RegisteredNode) (metadata.RegisteredNode, error)
+	PickNode(ctx context.Context, shardIDs []storage.ShardID, shardTotalNum uint32, registerNodes []metadata.RegisteredNode) (map[storage.ShardID]metadata.RegisteredNode, error)
+}
+type UniformityConsistentHashNodePicker struct {
+	logger *zap.Logger
 }
 
-type ConsistentHashNodePicker struct {
-	logger       *zap.Logger
-	hashReplicas int
+func NewUniformityConsistentHashNodePicker(logger *zap.Logger) NodePicker {
+	return &UniformityConsistentHashNodePicker{logger: logger}
 }
 
-func NewConsistentHashNodePicker(logger *zap.Logger, hashReplicas int) NodePicker {
-	return &ConsistentHashNodePicker{hashReplicas: hashReplicas, logger: logger}
+type nodeMember string
+
+func (m nodeMember) String() string {
+	return string(m)
```

Comment:
```suggestion
	return m
```

Reply:
This modification will cause compilation errors, golang compiler seems to be unable to recognize type alias normally.